### PR TITLE
Tne text

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Review by jobergum — APPROVED** [review]

**Comment by jobergum** [comment]
💯 🥳

**Review by kkraune — COMMENTED** [review]
👍

File: features.html [hunk 1]
```diff
@@ -42,20 +43,25 @@ <h2 id="query-capabilities">Query capabilities in Vespa</h2>
...
             <li><b><a href="https://docs.vespa.ai/en/reference/query-language-reference.html">Structured data</a></b>:
               Exact, substring and regexp, numerical ranges, geo-distance, predicates.</li>
             <li><b><a href="https://docs.vespa.ai/en/text-matching-ranking">Text</a></b>:
               Full text with tokenization and stemming, as well as positional information.</li>
-            <li><b><a href="https://docs.vespa.ai/en/nearest-neighbor-search.html">Vectors</a></b>:
-              Nearest neighbour, approximate (ANN) or exact, with a variety of distance metrics.</li>
           </ul>
+        <p>
+          A document can have any number of fields of these types that can be queried efficiently
+          in the same query, and each fields - even vector fields - can have multiple values in
```
Comment (geirst) [inline features.html:55]: "and each fields" -> "and each field"

Comment (bratseth) [reply features.html:55]: Thanks! https://github.com/vespa-engine/frontpage/pull/351

**Pull request merged**